checker/config.go

```diff
@@ -16,4 +16,6 @@ type Config struct {
 	LogFile                     string
 	LogLevel                    string
 	LogTriggersToLevel          map[string]string
+	MetricEventPopBatchSize     int
```

Comment:
let's add also in [config](https://github.com/moira-alert/moira/blob/master/pkg/notifier/notifier.yml), these two params please?)

Reply:
Okey